packages/react-scripts/template/README.md

```diff
@@ -327,10 +327,12 @@ Now we can make sure every file is formatted correctly by adding a few lines to
 Add the following line to `scripts` section:
 
 ```diff
++ "husky": {
```

Comment:
Can you move this below with the part about adding `lint-staged` to package.json? Something like: "Next we add 'lint-staged' and 'husky' fields to the `package.json`..."